_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -842,130 +857,683 @@ If you need to brush up on how the IaC Library works, read the
+##
 
-[[create_password_policy]]
-=== Create an IAM user password policy
-After the manual steps are complete, the next step is to create an IAM user password policy using the
-link:https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/examples/iam-password-policy/README.adoc[`iam-password-policy`
 wrapper module]. Complete this step before creating any IAM users!
+=== Configure the security baseline for the root account
+
+Next, we'll configure a security baseline for the root account that is responsible for creating all the child accounts.
+It will also configure AWS Organizations, IAM Roles, IAM Users, IAM Groups, IAM Password Policies, Amazon GuardDuty, AWS CloudTrail, AWS Config and Security Hub.
 
-[[wrapper_code]]
-==== Using the wrapper module
-This section demonstrates how to use the CIS compliance password policy wrapper module to create a password policy. Follow this pattern for each of the wrapper modules discussed throughout this walkthrough.
+We'll be using the `account-baseline-root` module from https://github.com/gruntwork-io/terraform-aws-cis-service-catalog[terraform-aws-cis-service-catalog].
 
-===== Using the wrapper module with Terraform
-If you're using Terraform without Terragrunt, use this section to create a module in your `infrastructure-modules` repository.
+[.exceptional]
+IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork subscriber# to access `terraform-aws-service-catalog`.
+
+First, create a _wrapper module_ called `account-baseline-root` in your `infrastructure-modules` repo under the `landingzone` subdirectory:
 
 ----
 infrastructure-modules
-└── security
-    └── iam-password-policy
-        ├── main.tf
-        └── variables.tf
+  └ landingzone
+    └ account-baseline-root
+      └ main.tf
+      └ outputs.tf
+      └ variables.tf
 ----
 
-Inside of `main.tf`, configure your AWS provider settings:
+Inside of `main.tf`, configure your AWS provider and Terraform settings:
 
-.infrastructure-modules/security/iam-password-policy/main.tf
+.infrastructure-modules/landingzone/account-baseline-root/main.tf
 [source,hcl]
 ----
 provider "aws" {
+  # The AWS region in which all resources will be created
   region = var.aws_region
+
+  # Require a 3.x version of the AWS provider
+  version = "~> 3.40"
+
+  # Only these AWS Account IDs may be operated on by this template
+  allowed_account_ids = [var.aws_account_id]
+}
+
+terraform {
+  # The configuration for this backend will be filled in by Terragrunt or via a backend.hcl file. See
+  # https://www.terraform.io/docs/backends/config.html#partial-configuration
+  backend "s3" {}
+
+  # Only allow this Terraform version. Note that if you upgrade to a newer version, Terraform won't allow you to use an
+  # older version, so when you upgrade, you should upgrade everyone on your team and your CI servers all at once.
+  required_version = "= 0.14.8"
```

Comment:
also upped the TF version here - this is what I've been testing with. Should it be something else? I assume once this repo is also upgraded to TF 0.15, we'll need to update this anyway

Reply:
Yep, leave it as 0.14.x for now and we can bump to 0.15 afterwards.